osu.Game/Screens/IOsuScreen.cs

```diff
@@ -21,15 +22,20 @@ public interface IOsuScreen : IScreen
         bool DisallowExternalBeatmapRulesetChanges { get; }
 
         /// <summary>
-        /// Whether the user can exit this <see cref="IOsuScreen"/> by pressing the back button.
+        /// Whether the user can exit this <see cref="IOsuScreen"/>.
         /// </summary>
-        bool AllowBackButton { get; }
+        /// <remarks>
+        /// When overriden to <c>false</c>,
+        /// the user is blocked from exiting the screen via the <see cref="GlobalAction.Back"/> action,
+        /// and the back button is hidden from this screen by the initial state of <see cref="BackButtonVisibility"/> being set to hidden.
+        /// </remarks>
+        bool AllowUserExit { get; }
 
         /// <summary>
         /// Whether a footer (and a back button) should be displayed underneath the screen.
         /// </summary>
         /// <remarks>
-        /// Temporarily, the back button is shown regardless of whether <see cref="AllowBackButton"/> is true.
+        /// Temporarily, the back button is shown regardless of whether <see cref="AllowUserExit"/> is true.
```

Comment:
Checking this behaviour, it looks back-to-front to me. Confused:

https://github.com/ppy/osu/blob/3e1b4f4ac564a1b69b2d8111b19d3908f99980e4/osu.Game/OsuGame.cs#L1617

Reply:
That comment meant the footer's back button, not our classic back button. I'll update the comment.